examples/layouts/basic-layout.html

```diff
@@ -44,4 +44,4 @@
   <script defer src="./common/map.js"></script>
 </body>
 
-</html>
+</html>
```

Comment:
👀

Reply:
same issue multiple files

Reply:
I think @VictorVelarde has a preset to remove trailing blank lines from files :(

I am gonna revert those changes.

Reply:
Sorry, I'll check it out. My user settings includes `"files.trimTrailingWhitespace": false` but maybe it is some other extension

Reply:
I've changed my VSCode user settings to this:
```
 "files.trimTrailingWhitespace": true,
 "files.insertFinalNewline": true,
 "files.trimFinalNewlines": true,
```
This guarantees one, and only one, new line at EOF